src/components/Navbar.css

```diff
@@ -1,20 +1,28 @@
 .navbar {
   background: rgb(233, 111, 37);
-  height: 80px;
   display: flex;
   justify-content: center;
   align-items: center;
   font-size: 1.2rem;
-  position: sticky;
-  top: -80px;
   z-index: 999;
   font-family: 'Montserrat', sans-serif;
   font-weight: 600;
   transition: top 1s;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  transition: height 500ms ease-in-out, opacity 400ms ease-in-out;
 }
 
 .hide-navbar {
-  top: 0;
```

Comment:
This particular line of code, paired with the `position:sticky` above, was the culprit for your navbar not appearing when you scrolled down. This was causing your navbar to be at the VERY TOP of the page, no matter where you scroll.